cross-chain/polygon/deploy_sidechain/05_upgrade_polygon_wormhole_gateway.ts

```diff
@@ -0,0 +1,50 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { ethers, getNamedAccounts, helpers, deployments } = hre
+  const { deployer } = await getNamedAccounts()
+
+  const PolygonTokenBridge = await deployments.get("PolygonTokenBridge")
+  const PolygonWormholeTBTC = await deployments.get("PolygonWormholeTBTC")
+  const PolygonTBTC = await deployments.get("PolygonTBTC")
+
+  const [, proxyDeployment] = await helpers.upgrades.upgradeProxy(
+    "PolygonWormholeGateway",
+    "PolygonWormholeGateway",
+    {
+      contractName:
+        "@keep-network/tbtc-v2/contracts/l2/L2WormholeGateway.sol:L2WormholeGateway",
+      initializerArgs: [
+        PolygonTokenBridge.address,
+        PolygonWormholeTBTC.address,
+        PolygonTBTC.address,
+      ],
+      factoryOpts: { signer: await ethers.getSigner(deployer) },
+      proxyOpts: {
+        kind: "transparent",
+      },
+    }
+  )
+
+  // Contracts can be verified on L2 Arbiscan in a similar way as we do it on
+  // L1 Etherscan
+  if (hre.network.tags.arbiscan) {
+    // We use `verify` instead of `verify:verify` as the `verify` task is defined
+    // in "@openzeppelin/hardhat-upgrades" to verify the proxy’s implementation
+    // contract, the proxy itself and any proxy-related contracts, as well as
+    // link the proxy to the implementation contract’s ABI on (Ether)scan.
+    await hre.run("verify", {
```

Comment:
Don't we need a modified verification script as we had in https://github.com/keep-network/tbtc-v2/blob/7f34311610593d6bf09d8b7fa96106678ab00766/cross-chain/polygon/deploy_sidechain/04_verify_polygon_wormhole_gateway.ts ?

Reply:
I think it's a nice to have and won't hurt us having it in the same file. Please see the logs above for Arbitrum. It will just say e.g. 
`The contract 0xA5daE77448ac98e908487ad0f7a511A9607d69a3 has already been verified`